connectors/qualys/configurations/lang_en.json

```diff
@@ -0,0 +1,42 @@
+{
+    "connection":{
+        "schedule": {

```

Comment:
This can be removed as well.

Reply:
Schedule field is required. The Qualys Agent will send collected information every 4 hours. With default options ( 5m,10m,...) we will get less updates from the agent, so we are configuring schedulers with minimum interval of 1h,2h,4h,6h,... to get reasonable data to import.